elastic/apm-server
# Add support for target service in metric aggregation

## Motivation/summary

- `service.target.{type, name}` should be indexed to ES for internal_metrics
- Metric aggregation should offer more granularity by utilising `service.target.{type, name}` fields
- Transaction intake API should accept `service_target_{type, name}` as input for working with dropped span stats 

Full spec list available at elastic/apm#627

## Checklist


## How to test these changes

1. Send API request to transaction intake API with `service_target_{type, name}` fields for the dropped span stats
2. Send API request to intake API with `resource` and `service.target.*` fields
3. Send API request to intake API with `resource` field and without `service.target.*` fields
4. Observe the output metrics to contain the appropriate `service.target.*` fields

## Related issues

Closes #7745
